app/routes/client_manager.js

```diff
@@ -53,32 +54,48 @@ class ClientManager {
       },
 
       message: async (ws, message, isBinary) => {
-        const json = JSON.parse(decoder.write(Buffer.from(message)));
-        const consoleMessage = JSON.parse(JSON.stringify(json));
+        try {
+          const json = JSON.parse(decoder.write(Buffer.from(message)));
+          const consoleMessage = JSON.parse(JSON.stringify(json));
 
-        if (
-          consoleMessage?.request?.user_login?.password ||
-          consoleMessage?.request?.user_create?.password
-        ) {
-          consoleMessage.request[Object.keys(json.request)[0]].password =
-            "********";
-        }
-        console.log(
-          `[ClientManager] ws on message (pid=${process.pid})`,
-          consoleMessage
-        );
+          if (
+            consoleMessage?.request?.user_login?.password ||
+            consoleMessage?.request?.user_create?.password
+          ) {
+            consoleMessage.request[Object.keys(json.request)[0]].password =
+              "********";
+          }
+          console.log(
+            `[ClientManager] ws on message (pid=${process.pid})`,
+            consoleMessage
+          );
 
-        const responseData = await PacketProcessor.processJsonMessageOrError(
-          ws,
-          json
-        );
+          const responseData = await PacketProcessor.processJsonMessageOrError(
+            ws,
+            json
+          );
 
-        if (responseData) {
-          try {
-            ws.send(JSON.stringify(responseData));
-          } catch (e) {
-            console.error("[ClientManager] connection with client ws is lost");
+          if (responseData) {
+            try {
+              ws.send(JSON.stringify(responseData));
+            } catch (e) {
+              console.error(
+                "[ClientManager] connection with client ws is lost"
+              );
+            }
           }
+        } catch (err) {
+          console.error("[ClientManager] ws on message error", err);
```

Comment:
let's also log here a packed which was sent by user, 

e.g.


somewhere at the top
```
const rawPacket = decoder.write(Buffer.from(message))
````

and then 

```
console.error("[ClientManager] ws on message error", err, rawPacket);
```